frontend/client/components/CreateFlow/Milestones.tsx

```diff
@@ -154,14 +161,23 @@ const MilestoneFields = ({
         allowClear={false}
         onChange={time => onChange(index, { ...milestone, dateEstimated: time.unix() })}
         disabled={milestone.immediatePayout}
-        disabledDate={current =>
-          current
-            ? current <
-              moment()
-                .subtract(1, 'month')
-                .endOf('month')
-            : false
-        }
+        disabledDate={current => {
+          if (!previousMilestoneDateEstimate) {
+            return current
+              ? current <
+                  moment()
+                    .subtract(1, 'month')
+                    .endOf('month')
+              : false;
+          } else {
+            return current
+              ? current <
+                  moment()
+                    .subtract(1, 'month')
+                    .endOf('month') || current < previousMilestoneDateEstimate
+              : false;
+          }
+        }}
```

Comment:
You can edit older ones and still end up with an invalid config. This is still definitely an improvement though. I'm personally not all that worried about this, so I'm fine to leave it as-is. But we could also yell at them in the validator.

Reply:
Hm, I could use more notes on how to reproduce.

We can circle back if it ends up being a problem, an Admin rejection will work as well.